core/src/main/java/org/infinispan/topology/ClusterTopologyManagerImpl.java

```diff
@@ -118,12 +118,12 @@ public void inject(Transport transport,
       this.cacheManagerNotifier = cacheManagerNotifier;
       this.cacheManager = cacheManager;
       this.timeService = timeService;
+      this.eventLog = eventLogManager.getEventLogger();
```

Comment:
I'll roll back my changes for now, since https://github.com/infinispan/infinispan/pull/4053 has a more comprehensive solution.